runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeConfig.java

```diff
@@ -112,7 +112,10 @@ default boolean highlightSearchMatch()
 		position = 8,
 		keyName = "geSearchMode",
 		name = "Search Mode",
-		description = "The search mode to use for the GE"
+		description = "The search mode to use for the GE<br>"
+			+ "Default - Matches exact text only<br>"
+			+ "Fuzzy Only - Matches inexact text such as 'sara sword'<br>"
+			+ "Fuzzy Fallback - Uses default search, falling back to fuzzy search if no results were found that way"
```

Comment:
From feedback on discord
```suggestion
			+ "Fuzzy Fallback - Uses default search, falling back to fuzzy search if no results were found"
```